scipy/signal/tests/test_signaltools.py

```diff
@@ -2080,9 +2081,12 @@ def filtfilt(self, zpk, x, axis=-1, padtype='odd', padlen=None,
             return sosfiltfilt(sos, x, axis, padtype, padlen)
 
     def test_basic(self):
+        atol = 1e-11
+        if platform.machine() == 'aarch64':
+            atol = 5.28e-11
         zpk = tf2zpk([1, 2, 3], [1, 2, 3])
         out = self.filtfilt(zpk, np.arange(12))
-        assert_allclose(out, arange(12), atol=1e-11)
+        assert_allclose(out, arange(12), atol=atol)
```

Comment:
So for clarity: changing this to `atol=.28e-11` and removing the changes further up